apb.yml

```diff
@@ -57,3 +58,60 @@ plans:
       type: enum
       enum: ['bearer', 'public']
       default: public
+  - name: custom
+    description: This plan can be used if images need to be customized
+    free: True
+    metadata:
+      displayName: User Specified
+      mobileclient_bind_parameters_data:
+        - '{"name": "CLIENT_ID", "value": "metadata.name", "type": "path"}'
+        - '{"name": "CLIENT_TYPE", "value": "public", "type": "default"}'
+      serviceinstance_bind_parameters_data:
+        - '{"name": "CLIENT_ID", "value": "metadata.name", "type": "path"}'
+        - '{"name": "CLIENT_TYPE", "value": "bearer", "type": "default"}'
+    parameters:
+    - name: custom_idm_image
+      title: Identity Management Container image
+      description: The image to use for the Identity Management Service (You shouldn't need to change this)
+      default: docker.io/jboss/keycloak:4.8.3.Final
+      type: string
+      required: True
+    - name: custom_postgresql_image
+      title: PostgreSQL Container Image
```

Comment:
"Container Image" here has both words capitalized, where the previous one just has "Container" capitalized. Should both be consistent? When I check my PRs for the other APBs, I've got both lower case :see_no_evil:

Reply:
right, ok, I will use lowercase then to be consistent with other fields